setup.py

```diff
@@ -15,13 +15,7 @@
 author = 'DabApps'
 author_email = 'contact@dabapps.com'
 license = 'BSD'
-install_requires = [
-    'django-model-utils==1.5.0',
-    'django-uuidfield==0.4.0',
-    'jsonfield==0.9.20',
-    'Django>=1.6',
-    'simplesignals==0.3.0'
-]
+install_requires = []
```

Comment:
Errrr I think you need this still? `pip install` won't read the `requirements.txt`